packages/google_sign_in/google_sign_in/ios/Classes/FLTGoogleSignInPlugin.modulemap

```diff
@@ -0,0 +1,10 @@
+framework module google_sign_in {
+  umbrella header "google_sign_in-umbrella.h"
+
+  export *
+  module * { export * }
+
+  explicit module Test {
```

Comment:
This isn't a private module since there are some complications for Swift files not being able to import those.

Instead, this explicit module requires that `google_sign_in.Test` be imported to access FLTGoogleSignInPlugin_Test.  So `initWithSignIn:` won't be available in the normal `import google_sign_in` case, won't auto-complete, etc.